src/ape/managers/config.py

```diff
@@ -69,6 +70,18 @@ def _plugin_configs(self) -> Dict[str, ConfigItem]:
         # Top level config items
         self.name = user_config.pop("name", "")
         self.version = user_config.pop("version", "")
+
+        if "contracts_folder" in user_config:
+            contracts_folder_value = Path(user_config.pop("contracts_folder")).expanduser()
+
+            # Attempt to resolve the path in the case it is relative to the project directory.
+            contracts_folder = Path(contracts_folder_value).resolve()
+            if not contracts_folder.exists():
+                raise ConfigError(f"Contracts path '{contracts_folder}' does not exist.")
```

Comment:
Note: this does not ensure there are actually contracts in the directory - it could be empty. It seems better to wait until the user tries to compile before doing that, which is why we still have a check i `ape_compile._cli.py::compile`

Reply:
HMMMMMMM

maybe it is just most consistent to _not_ have this check....